packages/components/src/ui/styles/hooks/use-rtl.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * @return {boolean} Whether the current document is RTL.
+ */
+export function useRTL() {
```

Comment:
We should remove this one I think, we rely on `wp.i18n.isRTL` concistently across our code base.